core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1167,24 +1172,24 @@ class EMConfig {
 
     enum class StoppingCriterion {
         TIME,
-        ACTION_EVALUATIONS
+        ACTION_EVALUATIONS,
+        INDIVIDUAL_EVALUATIONS
     }
 
     @Cfg("Stopping criterion for the search")
     var stoppingCriterion = StoppingCriterion.TIME
 
 
-    val defaultMaxActionEvaluations = 1000
+    val defaultMaxEvaluations = 1000
 
-    @Cfg("Maximum number of action evaluations for the search." +
+    @Cfg("Maximum number of action/individual evaluations for the search." +
```

Comment:
`Maximum number of action or individual evaluations (depending on chosen stopping criterion) for the search.`

Reply:
I made all the changes.